tests/unit/preprocessor/_time/test_time.py

```diff
@@ -13,7 +13,8 @@
 
 import tests
 from esmvaltool.preprocessor._time import (extract_month, extract_season,
-                                           time_average, extract_time)
+                                           time_average, extract_time,
+                                           _align_time_axes)
```

Comment:
use isort

Reply:
This is not done yet

Reply:
it is, the web editor is wonk, the code is:
```
from esmvaltool.preprocessor._time import (_align_time_axes, extract_month,
                                                                      extract_season, extract_time,
                                                                      time_average)
```

on a related topic it is worth telling people to keep isort updated since it is a tool outside the esmvaltool environment

Reply:
behh, don't mind the formatting on my code snippet, I am stoopid:
```
from esmvaltool.preprocessor._time import (_align_time_axes, extract_month,
                                           extract_season, extract_time,
                                           time_average)
```

Reply:
aaaand only now it's fixed :grin:

Reply:
> on a related topic it is worth telling people to keep isort updated since it is a tool outside the esmvaltool environment

Not if you install esmvaltool with `pip install -e .[develop]` as recommended in CONTRIBUTING.md